package.json

```diff
@@ -17,11 +17,13 @@
     }
   ],
   "peerDependencies": {
+    "circular-json": "^0.1.6",
```

Comment:
Because we do the obnoxious copying into `build/third_party` so we need to have any node_modules we use in the project using the code.